.github/workflows/CompatHelper.yml

```diff
@@ -12,5 +12,6 @@ jobs:
       - name: CompatHelper.main()
         env:
           GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-          # COMPATHELPER_PRIV: ${{ secrets.COMPATHELPER_PRIV }}  # optional
-        run: julia -e 'using CompatHelper; CompatHelper.main(; subdirs = ["", "test", "deps"])'
+          COMPATHELPER_PRIV: ${{ secrets.DOCUMENTER_KEY }}  # optional
+        run: julia -e 'using CompatHelper; CompatHelper.main(; subdirs=[""])'
+        # run: julia -e 'using CompatHelper; CompatHelper.main(; subdirs=["", "docs", "test"])'
```

Comment:
Why don't you include `docs` and `test`?

Reply:
Fixed in 29d626d.